src/shared/codelens/localLambdaRunner.ts

```diff
@@ -361,89 +356,71 @@ export const makeBuildDir = async (): Promise<string> => {
     return buildDir
 }
 
-export async function makeInputTemplate(params: {
-    baseBuildDir: string,
-    templateDir: string,
-    documentUri: vscode.Uri
-    originalHandlerName: string,
+export function getHandlerRelativePath(params: {
+    codeRoot: string,
+    filePath: string
+}): string {
+    return path.relative(params.codeRoot, path.dirname(params.filePath))
```

Comment:
Should this also include the filename without extension? I.e. if the `CodeUri` is `/mycode` and `filePath` is `/mycode/src/app.py`, we want to resolve to `src.app.myHandler`, not `src.myHandler`.

Reply:
It's not obvious to me what is the intended purpose of either of these functions (`getHandlerRelativePath` and `getRelativeFunctionHandler`). What if we replaced them with a single function like:

```typescript
export function getFullHandlerName({
    codeUri,
    sourceFilePath
}: {
    codeUri: string,
    sourceFilePath: string
}): string {
    ...
}
```

Reply:
I'll take a look, most of these weird workflows come from Python (and I'm assuming all other interpreted languages). I'm mainly building these methods and having everything go through the same workflows for continuity between run commands.